app/src/main/java/com/jwisozk/flightreviews/util/ParamEnum.kt

```diff
@@ -0,0 +1,11 @@
+package com.jwisozk.flightreviews.util
+
+enum class ParamEnum {
+    FLIGHT,
+    PEOPLE,
+    AIRCRAFT,
+    SEAT,
+    CREW,
+    FOOD,
+    TEXT
+}
```

Comment:
Minor stuff, but let's add a new line at the end of each file, either `ktlint` or `detekt` will trigger in this case